jetty-client/src/main/scala/org/http4s/jetty/client/JettyClient.scala

```diff
@@ -62,9 +62,10 @@ object JettyClient {
           })
         }
       )
-    val dispose = F
-      .delay(client.stop())
-      .handleErrorWith(t => F.delay(logger.error(t)("Unable to shut down Jetty client")))
+    val dispose =
+      F
+        .blocking(client.stop())
+        .handleErrorWith(t => F.delay(logger.error(t)("Unable to shut down Jetty client")))
     Resource.make(acquire)(_ => dispose)
```

Comment:
Kinda unrelated but shouldn't we dispose regardless of whether acquire succeeds?
```suggestion
    Resource.eval(acquire).onFinalize(dispose)
```

Reply:
Probably that would mean behavior change. According to the contract, `client` may come to the `resource` from outside.

Reply:
I guess we want to stop it if and only if we start it?

Reply:
Yeah, that makes sense.